Document users hashed passwords with argon2 does not work in FIPS env… (#33481)
keycloak/keycloak: docs/documentation/release_notes/topics/25_0_0.adoc

To prevent excessive memory and CPU usage, the parallel computation of hashes by Argon2 is by default limited to the number of cores available to the JVM.
To support the memory intensive nature of Argon2, we have updated the default GC from ParallelGC to G1GC for a better heap utilization.

> Note that Argon2 is not compliant with FIPS 140-2. So if you are in the FIPS environment, the default algorithm will be still PBKDF2. Also note that if you are on non-FIPS environment and

Reviewer: ```suggestion
Note that Argon2 is not compliant with FIPS 140-2. So if you are in the FIPS environment, the default algorithm will be still PBKDF2. Also note that if you are on non-FIPS environment and
```

> you plan to migrate later to the FIPS environment, it can be good idea to explicitly switch password policy to some FIPS compliant algorithm like `pbkdf2-sha512` from the beginning as otherwise

Reviewer 2: ```suggestion
you plan to migrate later to the FIPS environment, it can be good idea to explicitly switch the password policy to some FIPS compliant algorithm like `pbkdf2-sha512` from the beginning as otherwise
```

> you plan to migrate later to the FIPS environment, it can be good idea to explicitly switch the password policy to some FIPS compliant algorithm like `pbkdf2-sha512` from the beginning as otherwise
> your users will not be able to login after the switch to the FIPS environment.

Reviewer: ```suggestion
you plan to migrate to the FIPS environment, consider changing the password policy to a FIPS compliant algorithm such as `pbkdf2-sha512` at the outset. Otherwise, users will not be able to log in after they switch to the FIPS environment.
```
